### PR TITLE
Add tianshou

### PR DESCRIPTION
**Pull request by Trinkle23897**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/tianshou/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  sha256: 78403c7b66c11165460d244811573dd640abcde4e322f79fe4a35dc89602329c
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python
```
Comment (Trinkle23897) [inline recipes/tianshou/meta.yaml:19]:
```suggestion
    - python >=3.6
```

File: recipes/tianshou/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+test:
+  imports:
+    - tianshou
+
+about:
+  home: https://github.com/thu-ml/tianshou
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'Simple, fast, extensible JSON encoder/decoder for Python'
```
Comment (Trinkle23897) [inline recipes/tianshou/meta.yaml:33]:
```suggestion
  summary: 'An elegant, flexible, and superfast PyTorch deep reinforcement learning platform.'
```

File: recipes/tianshou/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+test:
+  imports:
+    - tianshou
+
+about:
+  home: https://github.com/thu-ml/tianshou
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
```
Comment (synapticarbors) [inline recipes/tianshou/meta.yaml:37]: Please add a comment above the `license_file` line that says something like:

```
# License file manually packaged. See https://github.com/thu-ml/tianshou/pull/227`
```

I opened up a PR (https://github.com/thu-ml/tianshou/pull/227) to include the license file in the source distribution. If that PR is merged or some other solution is implemented, then in future releases, this acts as a reminder to use the copy of the license file that comes in the source distribution.

**Review by synapticarbors — COMMENTED** [review]
One small request, but should be good to merge after that.

File: recipes/tianshou/meta.yaml (hunk 3 above)
Comment (Trinkle23897) [reply recipes/tianshou/meta.yaml:37]:
```suggestion
    # License file manually packaged. See https://github.com/thu-ml/tianshou/pull/227
    license_file: LICENSE
```

Comment (Trinkle23897) [reply recipes/tianshou/meta.yaml:37]: Should I remove the LICENSE in this PR?

**Comment by synapticarbors** [comment]
You can't remove the manually included license here until a new release is made that include the license file in the source distribution.

**Comment by Trinkle23897** [comment]
> You can't remove the manually included license here until a new release is made that include the license file in the source distribution.

Done!

**Comment by synapticarbors** [comment]
@Trinkle23897 -- can you give an example of one that you've seen? There are a few different ways to include the license file in the source distribution. The `MANIFEST.in` I think works most broadly, but there are other ways including entries in the `setup.py` or `setup.cfg`.

**Comment by Trinkle23897** [comment]
> The `MANIFEST.in` I think works most broadly, but there are other ways including entries in the `setup.py` or `setup.cfg`.

Thank you for your declaration!

**Pull request merged**